src/sentry/projects/project_rules/creator.py

```diff
@@ -32,8 +32,14 @@ def run(self) -> Rule:
             if features.has(
                 "organizations:workflow-engine-issue-alert-dual-write", self.project.organization
             ):
-                # TODO(cathy): handle errors from broken actions
-                migrate_issue_alert(self.rule, self.request.user.id if self.request else None)
+                try:
+                    with transaction.atomic(router.db_for_write(Rule)):
```

Comment:
i decided not to add a nested transaction for update (which has a similar structure). we don't want the Rule and Workflow to be out of sync